a/ambondrona.md

```diff
@@ -0,0 +1,9 @@
+# Ambondrona
+
+![Ambondrona](ambondrona.jpg)
+
+AmbondronA is a pop rock from Madagascar. The band has figured prominently in the Malagasy music scene since 2001. AmbondronA has toured extensively throughout Madagascar, and has given numerous performances throughout the Indian Ocean Islands, France and South Africa. The band has released six albums. Wikipedia
```

Comment:
AmbondronA is a pop rock *band* from Madagascar
Please remove the trailing `Wikipedia`